packages/nds-hero/src/Hero.js

```diff
@@ -4,57 +4,37 @@ import PropTypes from "prop-types";
 import "../scss/hero.scss";
 
 export const Hero = props => {
+	const { actions, children, footer, header, intro, title } = props;
+	let Footer;
+	if (footer && footer.content) {
+		Footer = footer.elementType || "div";
+	}
 	return (
 		<div className="hero">
 			<div className="hero__container">
+				{header && header}
 				<div className="hero__body">
 					<div className="hero__copy">
-						<h1 className="hero__title">Hero title</h1>
-						<p className="hero__intro">
-							This is a hero intro that should explain in a few words what the
-							site is about
-						</p>
-						<div className="hero__actions">
-							<a
-								href="/nice-design-system/about/getting-started/"
-								className="btn btn--cta"
-							>
-								Primary CTA
-							</a>
-							<a
-								href="https://github.com/nhsevidence/nice-design-system"
-								className="btn"
-								target="_blank"
-								rel="noopener noreferrer external"
-							>
-								Secondary cta
-							</a>
-						</div>
-					</div>
-					<div className="hero__extra">
-						<h2 className="h4 mt--0-md">Quick links</h2>
-						<ul className="list list--unstyled list--loose">
-							<li>
-								<a href="/nice-design-system/foundations/">Show links here</a>
-							</li>
-							<li>
-								<a href="/nice-design-system/components/">Maximum of 6</a>
-							</li>
-							<li>
-								<a href="/nice-design-system/technical/">
-									Show the most frequent
-								</a>
-							</li>
-						</ul>
+						<h1 className="hero__title">{title}</h1>
+						{intro && <p className="hero__intro">{intro}</p>}
+						{actions && <div className="hero__actions">{actions}</div>}
 					</div>
+					{children && <div className="hero__extra">{children}</div>}
 				</div>
+				{footer && <Footer className="hero__footer">{footer.content}</Footer>}
```

Comment:
I think this makes for an easier API if footer is just a prop as a straight up React.Node that lives inside a a `div.hero__footer` so you don't need to pass an object in